packages/fiori/src/ShellBarItem.ts

```diff
@@ -66,6 +69,9 @@ class ShellBarItem extends UI5Element {
 	@property()
 	count?: string;
 
+	@property({ type: Object })
```

Comment:
JSDoc can be added to describe the possible entries, like the one inside shellbar